solutions/gold/cf-912E.mdx

```diff
@@ -0,0 +1,96 @@
+---
+id: cf-912E
+source: CodeForces
+title: Prime Gift
+author: Daniel Ge
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/56920)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(\log 10^{18} \cdot (|D(A)| + |D(B)|))$
```

Comment:
constant factors are omitted from big O right?

maybe add an `<Asterisk>` about it or a small note if it contributes a huge chunk of it?